SConscript

```diff
@@ -48,6 +48,8 @@ if env['noadaptivesync']==1:
     env.Append(CPPDEFINES    = 'NOADAPTIVESYNC')
 if env['l2_security']==1:
     env.Append(CPPDEFINES    = 'L2_SECURITY_ACTIVE')
+if env['single_ch_slot']!=0:
```

Comment:
Need to guarantee the value is valid, such as 11-26

Reply:
That would only apply for 2.4 GHz, is that OK?

Reply:
Yeah, I think that is fine, we're currently only operating in the 2.4 Ghz band. I don't think the subGhz radio already works on the OpenMote-B's.